examples/text-generation/README.md

```diff
@@ -98,8 +98,8 @@ Here are a few settings you may be interested in:
 - `--prompt` to benchmark the model on one or several prompts of your choice
 - `--attn_softmax_bf16` to run attention softmax layer in bfloat16 precision provided that the model (such as Llama) supports it
 - `--trim_logits` to calculate logits only for the last token in the first time step provided that the model (such as Llama) supports it
-- `--kv_cache_fp8` Store kv-cache in float8 when kv-cache is used
 - `--fp8` Enable Quantization to fp8
+- `--kv_cache_fp8` Deprecated - Store kv-cache in float8 when kv-cache is used. should not be used with HQT(Habana Quantization Toolkit)
```

Comment:
per business, we should not use "Habana Quantization Toolkit". we agreed to use "The Quantization Toolkit". this is aligned with RTD